README.md

```diff
@@ -33,99 +33,45 @@ This package supports dbt version `1.2.x` or higher.
 
 ### Step 1: Map Your Claims Data to the Tuva Claims Data Model
 
+See our [Quickstart Guide](https://thetuvaproject.com/quickstart) for detailed instructions.
+
 The first step is mapping your claims data to the Tuva Claims Data Model.  You can map your claims data to the Tuva Claims Data Model yourself (i.e. by writing SQL inside your dbt project).  Or if you have Medicare CCLF or Medicare SAF (LDS) claims data you can use our connectors, which are separate repos that you can find on our GitHub page.  You need to create each of the tables in the Tuva Claims Data Model as models within your dbt project so that the Tuva Project dbt package can reference them using ref() functions.
```

Comment:
Are we calling it the "Tuva Claims Data Model" or the "Tuva Input Layer"? The latter is how they can find the data dictionary in the docs.

Reply:
All of that should really change... We have a new tiny mart currently called "Input Layer" which selects from the User Defined Models and explicitly casts each column to the correct data type. Its the source of truth for the table definitions they need to map to and is driving the column definitions Input Layer section of the docs site, but technically the input layer isn't what they're building, they're building the input to the input layer.

This is not clear in the docs... the docs makes it seem like you map to the input layer.  I would be confused as a user.

Not going to try and solve that in the readme just yet. tagging @aneiderhiser